core/domain/user_query_domain_test.py

```diff
@@ -22,17 +22,60 @@
 
 from core import feconf
 from core import utils
+from core.constants import constants
 from core.domain import user_query_domain
 from core.tests import test_utils
 
+from typing import TYPE_CHECKING
+
+
+class UserQueryParamsAttributeTests(test_utils.GenericTestBase):
+    """Test for ensuring matching values for UserQueryParams attributes between
+    predefined and dynamically fetched fields from assets/constants.ts
+    """
+
+    def test_matching_values_for_user_query_params_attributes(self) -> None:
+        """Check to see if list of attributes defined here
+        (attribute_names_predefined) is similar to the one we get during runtime
+        (attribute_names)from (/assets/constants.ts)
+        """
+
+        attribute_names = [
+                predicate['backend_attr'] for predicate in (
+                    constants.EMAIL_DASHBOARD_PREDICATE_DEFINITION)]
+
+        if not TYPE_CHECKING:
```

Comment:
Do we need this? I would rather use mypy ignores here.

Reply:
ohh my bad !! I used this earlier and forgot to remove it . Yes we can remove it entirely . There's no need for it here !!